arch/xtensa/core/fatal.c

```diff
@@ -11,6 +11,10 @@
 #include <misc/printk.h>
 #include <xtensa/specreg.h>
 
+#ifdef XT_SIMULATOR
+#include "simcall.h"
+#endif
+
```

Comment:
Why does this need to be added?  The code in this file that is predicated on XT_SIMULATOR doesn't seem to require any external symbols.

Reply:
```SYS_exit``` defined in simcall.h is used in fatal.c [here]( https://github.com/GautierAtWork/zephyr-pub/blob/56caa360a1b98b10021770b64af5f618110cb073/arch/xtensa/core/fatal.c#L211)